src/styles/mixins.less

```diff
@@ -6,14 +6,13 @@
 	max-height: @max;
 }
 
-// gradient animation mixin
-// takes start color, end color and transition time (set to .1s by default)
-.gradient-animation(@start, @end, @transTime: .1s, @zindex: 0) {
+// Gradient animation mixin -- basically used for buttons
+// takes a start and end color
+.gradient-animation(@start, @end) {
```

Comment:
Nothing using `.gradient-animation` was using the last two arguments, so I axed them.